src/components/modal/modal.tsx

```diff
@@ -103,9 +105,13 @@ export const Modal: FC<ModalProps> = ({
             transition={{ duration: 0.3 }}
           >
             <ModalHeader title={title} headerNode={headerNode} onClose={closeModal} />
-            <Scrollbars autoHeight autoHeightMax={contentMaxHeight} hideTracksWhenNotNeeded>
+            {scrollable ? (
+              <Scrollbars autoHeight autoHeightMax={contentMaxHeight} hideTracksWhenNotNeeded>
```

Comment:
what about incupsulation this logic into a separate component? 
```
{scrollable 
? <ScrollableModalContent>{children}</ScrollableModalContent> 
: <ModalContent>{children}</ModalContent>
}
```

Reply:
Agree, it would be nice to store the size/etc. calculations inside this component as well.
I suggest to postpone this improvement until the ui-kit is integrated with the RP UI as the team plans a similar improvement.